NEWS

```diff
@@ -12,6 +12,7 @@ Noteworthy changes in release ?.?? (????-??-??)
   * Implemented decoding of PR_SET_VMA operation of prctl syscall.
   * Updated lists of FAN_*, IORING_*, IOSQE_*, KVM_*, MODULE_INIT_*, TCA_ACT_*,
     and *_MAGIC constants.
+  * Add decoding of si_pkey field in siginfo struct
```

Comment:
Please make sure each sentence ends with a dot.

If you have a look at the NEWS file, you'll see that almost every
"Improvements" section ends with "Updated lists of constants" line followed by
"Updated lists of ioctl commands" line.
Let's keep this tradition, please add your line before rather than after
these lines.